ifelse/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	if 8%2 == 0 {
+		fmt.Println("Output Is Even")
+	} else {
+		fmt.Println("Output Is Odd")
+	}
+	if 10%5 == 2 {
+		fmt.Println("Remainder  is 2")
```

Comment:
This needs to be 0 and not 2
- It is `the remainder` and not `the quotient`
- You are using `%` which gives remainder and since `10%5 = 0` , you have remainder as 0 where as your quotient is 2

Reply:
@atharvaskadam bro don't forget to mark comments as resolved